src/synthgauge/evaluate.py

```diff
@@ -16,7 +18,24 @@
 
 
 class Evaluator(object):
-    """ """
+    """
+    The central class in ``synthgauge`` used to hold and evaluate data via
+    metrics or visualisation.
```

Comment:
```suggestion
    metrics and visualisation.
```
Before it read like you had to choose.

Reply:
A fair point... but what about all the logicians out there grumbling about the difference between "OR" and "XOR" operators?